src/selenium/basePage.js

```diff
@@ -0,0 +1,483 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const until = require('selenium-webdriver').until;
+const By = require('selenium-webdriver').By;
+const config = require('../../config.json');
+const request = require('request').defaults({'proxy': config.proxy});
+
+const BasePage = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement. (one-var)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)